release-notes/major9/09.26.0/valtimo-frontend-libraries.md

```diff
@@ -71,4 +72,4 @@ This version has the following known issues:
 
 * **Issue2**
   * Discovered in version x.x.x
-  * Describe what can be done to work around the issue
\ No newline at end of file
+  * Describe what can be done to work around the issue
```

Comment:
?

Reply:
A newline was added to the end of the file